src/Dialogs/Settings/Panels/LayoutConfigPanel.cpp

```diff
@@ -186,6 +187,14 @@ LayoutConfigPanel::Prepare(ContainerWindow &parent, const PixelRect &rc)
           _("A list of possible InfoBox layouts. Do some trials to find the best for your screen size."),
           info_box_geometry_list, (unsigned)ui_settings.info_boxes.geometry);
 
+  AddInteger(_("InfoBox title size"),
+             _("An InfoBox has 3 lines: the value in the middle, "
+               "the title in the top line and the comment in the bottom line. "
```

Comment:
I don't think this should be explained here. This only adds work for translators, translating redundant text, and users will not read it anyway because it's too long.

Reply:
I'll move this text to the documentation.